platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -553,6 +553,30 @@ check_shared_library_dependency_display_util() {
     esac
 }
 
+check_package_manager() {
+    not_found=0
+    package_managers="apt-get dpkg yum rpm"
+    for package in $package_managers; do
+    {
+        res="$(need_cmd $package)"  
+        if [ $? -eq 0 ] ; then
+            not_found=0
+            wrap_debug "$package package manager is present."
+            break;
+        else
+            not_found=1
+            wrap_debug "$package package manager is not present."
+        fi
+    }
+    done
+    if [ "$not_found" -eq 1 ]; then
+        wrap_warning "IoT Edge supports the following package types [*deb, *rpm] and following package managers [apt-get].We have identified that this device does not have support for the supported package type. Please head to aka.ms/iotedge for instructions on how to build the iotedge binaries from source"
+        if [ ! -d "/etc/ca-certificates" ]; then
+            wrap_warning "Install CA certificates package" 
+        fi
```

Comment:
We should do a 
wrap_pass if our checks suceeded
OR 
wrap_warn if they didnts (Since this is a soft check)